publish/release.ps1

```diff
@@ -132,6 +134,27 @@ Publish-Module -Path $psGalleryDir -NuGetApiKey $PsGalleryApiKey -Verbose -Force
 if (-not $isPreRelease -or $Force) {
     & nuget push $nupkg -Source https://api.nuget.org/v3/index.json -apikey $NugetApiKey
     & nuget push $nupkg -Source https://push.chocolatey.org/ -apikey $ChocolateyApiKey
+
+    if ([string]::IsNullOrWhiteSpace($GitHubUsername) -or [string]::IsNullOrWhiteSpace($GitHubToken)) {
+        Write-Host "Missing GitHub credentials, not triggering a Pester documentation update for $version"
+    } else {
+        $encodedCreds = [System.Convert]::ToBase64String([System.Text.Encoding]::ASCII.GetBytes("$($GitHubUsername):$($GitHubToken)"))
+        $Headers = @{
+            'Accept' = 'application/vnd.github.v3+json'
+            'Authorization' = "Basic $encodedCreds"
+        }
+        $Body = @{
+            'ref' = 'master'
+            'inputs' = @{
+                'pester-version' = $Version
+            }
+        }
+        # There can be caching issues between the time a packaged is pushed to the PS Gallery, and when it becomes searchable. Hopefully waiting one
+        # minute gives the Gallery enough time to process the new package and make it available
+        Start-Sleep -Seconds 60
```

Comment:
```suggestion
        Write-Host "Waiting 1 minute till package is on PSGallery"
        Start-Sleep -Seconds 60
```
Gimme some message so I know it's not stuck. 

Maybe polling using HEAD request after 20s on PSGallery and then every second till the resource is available would be better.